nodeJS/express/installing_postgresql.md

```diff
@@ -52,7 +52,7 @@ sudo apt install postgresql postgresql-contrib libpq-dev
 After installation is complete, let's start the server using this command:
 
 ```bash
-sudo systemctl start postgresql.service
+sudo systemctl start postgresql.service && systemctl status postgresql.service
```

Comment:
the status command should be mentioned separately. what do you think?

Reply:
got it, thanks.

You'll also need to update the rails version of the lesson here: https://github.com/TheOdinProject/curriculum/blob/main/ruby_on_rails/advanced_forms_and_activerecord/installing_psql.md

Mind making that change as well?